### PR TITLE
Update pronom container signatures 20180920

### PR DESCRIPTION
**Pull request by ablwr**
Related to #139. This stops the traceroute error and handles the exception, although I only tested it on a rogue file that may just intentionally be throwing an exception. I'll continue this work/review tomorrow!

### CODE REVIEW

File: fido/conf/versions.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <versions>
 	<pronomVersion>94</pronomVersion>
 	<pronomSignature>formats-v94.xml</pronomSignature>
-	<pronomContainerSignature>container-signature-20180917.xml</pronomContainerSignature>
+	<pronomContainerSignature>container-signature-20160121.xml</pronomContainerSignature>
```
Comment (mistydemeo) [inline fido/conf/versions.xml]: Is this the right version? This looks like a downgrade.

Comment (ablwr) [reply fido/conf/versions.xml]: Whoops -- the comment in updatesignatures.py#37 let me know this is hardcoded and has to be manually changed, and I missed it! Thank you!

**Comment by ablwr** [comment]
@mistydemeo You can review again if you'd like, although you're under no obligation! ;)

**Comment by ablwr** [comment]
:D :D :D

**Pull request merged**